front/app/containers/IdeasShow/components/RightColumnDesktop/index.tsx

```diff
@@ -103,14 +89,14 @@ const RightColumnDesktop = ({
                 <ProposalInfo idea={idea} />
               </Box>
             )}
-            {showReactionControl && (
-              <Box pb="23px" mb="23px">
+            {showIdeaReactionControl && (
+              <Box pb="23px" mb="23px" borderBottom="solid 1px #ccc">
                 <ReactionControl styleType="shadow" ideaId={ideaId} size="4" />
               </Box>
             )}
-            {phase && ideaIsInParticipationContext && (
+            {phase && ideaIsInParticipationContext && votingConfig && (
               <Box pb="23px" mb="23px" borderBottom="solid 1px #ccc">
-                {votingConfig?.getIdeaPageVoteInput({
+                {votingConfig.getIdeaPageVoteInput({
```

Comment:
Changed this slightly as I noticed that when there was no voting config AND no reactions there was a rogue line appearing above the comment button.